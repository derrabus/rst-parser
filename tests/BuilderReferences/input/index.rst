References
==========

Test :ref:`referencing a document <reference>`

Test :ref:`referencing an anchor in another document <test_anchor>`

Test referencing a `Sub section`_ and `Some Sub Section`_ (invalid)

Sub section
-----------
