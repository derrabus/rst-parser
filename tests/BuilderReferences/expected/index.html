<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8" />
    </head>
    <body>
        <div class="section" id="references">
            <h1>
                References
            </h1>
            <p>Test <a href="reference.html">referencing a document</a></p>
            <p>Test <a href="reference.html#test_anchor">referencing an anchor in another document</a></p>
            <p>Test referencing a <a href="index.html#sub-section">Sub section</a> and Some Sub Section (invalid)</p>
            <div class="section" id="sub-section">
                <h2>
                    Sub section
                </h2>
            </div>
        </div>
    </body>
</html>
